src/cf.data.pre

```diff
@@ -8704,51 +8704,6 @@ DOC_START
 	By default, Squid doesn't listen on any multicast groups.
 DOC_END
 
-NAME: mcast_miss_addr
```

Comment:
Removed directives should have entries added to the top of the file in a group for Squid-7 with type `obsolete`. See the directives already there for examples.